pkg/trace/config/config.go

```diff
@@ -85,6 +85,9 @@ type ObfuscationConfig struct {
 	// ES holds the obfuscation configuration for ElasticSearch bodies.
 	ES obfuscate.JSONConfig `mapstructure:"elasticsearch"`
 
+	// OpenSearch holds the obfuscation configuration for OpenSearch bodies.
+	OpenSearch obfuscate.JSONConfig `mapstructure:"opensearch"`
```

Comment:
I think we also need some changes in `comp/trace/config/setup.go` to setup the configuration based on the incoming yaml (see the example on line 411 from elasticsearch)

Reply:
Thanks for the pointer. I updated the setup based on the config.